src/hci/gnomex/controller/GetRequestDownloadList.java

```diff
@@ -725,11 +725,14 @@ private void recurseAddFiles(Element fdNode, File f1, String requestNumber, Stri
 			fileNode.setAttribute("state", "unchecked");
 			fileNode.setAttribute("canDelete", "Y");
 			fileNode.setAttribute("canRename", "Y");
-			//fdNode.setAttribute("linkedSampleNumber", getLinkedSampleNumber(sess, fileName.substring(fileName.indexOf(Request.getBaseRequestNumber(requestNumber)))));
+			StringBuffer fname = new StringBuffer();
+			fname.append(fullPath.substring(fullPath.indexOf(Request.getBaseRequestNumber(fd.getNumber()))).replace("\\","/"));
+			fname.append(f.getName());
+			fileNode.setAttribute("linkedSampleNumber", getLinkedSampleNumber(sess, fname.toString()));

```

Comment:
Tim,
I found that the method getLinkedSampleNumber expects the format of experimentNumber/directory (if there is one)/name of file.

Maybe I’m missing something.

Thanks,
-John